doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/config.md

```diff
@@ -258,12 +230,86 @@ The field `rayStartParams.resources` should only be used for custom resources. T
 `CPU`, `GPU`, and `memory` are forbidden. If you need to specify overrides for those resource
 fields, use the Ray start parameters `num-cpus`, `num-gpus`, or `memory`.
 
+(kuberay-networking)=
+## Services and networking
+### The Ray head service.
+The KubeRay operator automatically configures a Kubernetes Service exposing the default ports
+for several services of the Ray head pod, including
+- Ray Client (default port 10001)
+- Ray Dashboard (default port 8265)
+- Ray GCS server (default port 6379)
+- Ray Serve (default port 8000)
+
+The name of the configured Kubernetes Service is the name, `metadata.name`, of the RayCluster
+followed by the suffix\
```

Comment:
The `\` should be removed from here, or there will be a line break in the rendered documentation. Same 2 lines below :)

Reply:
I did this deliberately to prevent a highlighted technical term from being split between two lines.

This approach does not mesh well with subsequent edits to a doc, so let me consider alternatives...

Reply:
TIL, there's an html tag for that,  <nobr>